js/sales.js

```diff
@@ -123,6 +123,65 @@
         return dataCell;
     }
 
+    function getNewStore(event){
+        event.preventDefault();
+        const form = event.target;
+        const storeName = form.querySelector('#store-name').value;
+        const minHourlyCustomers = parseInt(form.querySelector('#min-hourly-customer').value);
+        const maxHourlyCustomers = parseInt(form.querySelector('#max-hourly-customer').value);
+        const avgCookiesPerCustomer = parseInt(form.querySelector('#avg-cookies-per-customer').value);
+        const newStore = new CookieStore(storeName, minHourlyCustomers, maxHourlyCustomers, avgCookiesPerCustomer);
+        cookieStoreLocations.push(newStore);
+        addNewStore(newStore);
+        form.querySelector('input[type=\'submit\']').disabled = true;
+        Array.from(form.querySelectorAll('input')).forEach(input => {
+            if(input.value !== 'Enter store'){
+                input.value = '';
+            }
+        });
+    }
+
+    function toggleTotalRowHighlight(){
+        if(cookieStoreLocations.length % 2 === 0){
+            document.querySelector('tfoot').removeAttribute('class', 'highlight');
+        } else {
+            document.querySelector('tfoot').setAttribute('class', 'highlight');
+        }
+    }
+
+    function newStoreNameInputValidation(event){
+        const storeNameInput = event.target;
+        if(!isNaN(storeNameInput.value)){
+            storeNameInput.value = '';
+            storeNameInput.setAttribute('class', 'name-warning-input');
+            document.querySelector('#hide-name-warning-message').setAttribute('id', 'name-warning-message');
+        } else {
+            storeNameInput.removeAttribute('class', 'name-warning-input');
```

Comment:
You might want to look into [the classList property](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) that all elements have and make use of the [add](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add), [remove](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/remove), or maybe even the [toggle](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle) methods. So then it would be something like `storeNameInput.classList.remove('name-warning-input');`.